xpdan/999-load.py

```diff
@@ -1,6 +1,91 @@
+import os
 import sys
+from mock import MagicMock
 from PyQt4 import QtGui
 app = QtGui.QApplication(sys.argv)
-from xpdan.qt_gui import XpdanSearch, an
 
+import matplotlib
+matplotlib.use('qt4agg')
+import tempfile
+import os
+import tzlocal
+from bluesky import RunEngine
+from bluesky.plans import scan, count, relative_scan
+from bluesky.examples import Reader, motor
+
+# set env_var
+# expected internal code: 
+# 0 -> beamline
+# 1 -> test
+# 2 -> simulation
+os.environ['ENV_VAR'] = str(2)
+
+from xpdan.glbl import an_glbl
```

Comment:
We might make the global factory take in an arg which specifies the type of setup to do. That way we can specify the type of setup even without an env_var.